keras_cv/layers/preprocessing/README.md

```diff
@@ -0,0 +1,66 @@
+# Preprocessing Layers
+
+KerasCV offers a bunch of pre-processing layers.
```

Comment:
KerasCV offers many preprocessing and data augmentation layers which support classification, object detection, and segmentation masks. When you use KerasCV augmentation layers to augment your training data, class labels, bounding boxes, and mask labels automatically get augmented alongside the image augmentations!